test/unit/helpers/ConfigHelper.ts

```diff
@@ -244,8 +251,15 @@ test(`promptOptions 9 - autoRegister= true backwards compatibility for custom in
       autoPrompt: false,
     },
     slidedown: {
-      enabled: true,
-      autoPrompt: false,
+      prompts: [{
```

Comment:
Should keep the old format so we are still testing what could be in the wild as is.

Can use `getPromptOptionsFromFinalAppConfig` to make the tests pass.